docs/source/run/install.rst

```diff
@@ -0,0 +1,139 @@
+
+.. warning::
+
+    Although possible, source installations are not supported. If you install
+    from sources, you may encounter unexpected bugs and unstable behaviors, and
+    we fully expect you to either find a solution by yourself, revert to a
+    previous version, or wait for a fix without an ETA.
+
+    However, we do appreciate :ref:`bug reports<faq-bug-report>` (with logs and
+    configuration details) and feedback. In that case,
+    :ref:`reaching out to us on IRC<faq-contact-us>` is the best approach.
+
+.. __: https://github.com/sopel-irc/sopel
+
+
+First run
+=========
+
+Once Sopel is installed, you should have access to a ``sopel`` command, as well
+as :doc:`other commands <cli>`.
+
+.. note::
+
+    By default, ``pip`` will install the command in ``~/.local/bin/``, which
+    might not be on your ``PATH``. You will need to add this folder to your
+    ``PATH`` for your shell to see the command.
+
+    For example, you could add this to your ``~/.profile`` file (works on
+    Ubuntu 22.04)::
+
+        # set PATH so it includes user's private bin if it exists
+        if [ -d "$HOME/.local/bin" ] ; then
+            PATH="$HOME/.local/bin:$PATH"
+        fi
+
+    Refer to your operating system's documentation for more information on how
+    to configure your ``PATH``.
+
+Initial Configuration
+---------------------
+
+To run the bot, you need a :doc:`configuration file <configuration>`, which you
+can create with the following command::
+
+    sopel configure
+
+This will run a wizard that helps you create a configuration file. It will
+ask you questions to fill in the details. When you see something in ``[square
+brackets]``, that's the default setting, and you can just hit Enter to keep
+it. We recommend selecting a custom nick for your bot during configuration.
+
+This wizard doesn't cover every option, only the ones which are needed to get
+the bot running. The :doc:`core config settings <configuration>` are all
+documented if you want to make other tweaks.
+
+Finally, the wizard will ask you about configuration settings for plugins. This
+will automatically detect what plugins you have available, and run their
+configuration utility if they have one.
+
+Once you are done, you can always re-run the same command and accept the values
+you already set and change only what you need.
+
+Start the Bot
+-------------
+
+Once you have properly configured your bot, you are now ready to start it for
+the first time. The command to do so is the following::
+
+    sopel start
+
+.. note::
+
+    Some IRC networks require nick registration or special configuration
+    for connecting. Refer to your IRC network for more information about
+    what steps may be necessary to run your bot there.
+
+Sopel will output log information informing you of its startup progress, such
+as loading plugins, connecting to the network, and joining channels.
+
+To stop the bot, simply exit the process (e.g. with :kbd:`Control+c`) and
```

Comment:
Renders as:
<img width="145" alt="image" src="https://github.com/sopel-irc/sopel/assets/164140/16e0f52b-ad63-45cc-b616-f5cf6d402509">

I'm happy with this experiment. It looks great! 😁